README.md

```diff
@@ -119,4 +119,7 @@ doesn't match w/ the current homepage. Maybe keep it as mynt though
 - but your choice completely.
 
 
--your friendly administrator.
+-your friendly administrator.  
+
+## LICENSE
+![License: Zlib](https://img.shields.io/badge/License-Zlib-lightgrey.svg)
```

Comment:
Hi @azkarmoulana, Thanks for the PR. A few suggestions:

- [ ] It's good practice in Markdown to have a blank line after all headings.
- [ ] In addition to the badge, it would be a good idea to link to the LICENSE file at the root of the repo.

Thanks!

Reply:
@willingc changes accomadated